README.md

```diff
@@ -30,6 +32,66 @@ Now run `jupyter notebook`, and it should have a Swift kernel.
 
 # Usage Instructions
 
+## Rich output
+
+You can call Python libaries using [Swift's Python interop] to display rich
```

Comment:
Did you intend `[Swift's Python interop]` to be a link? Same with `[Swift for Tensorflow]` below.

Reply:
it is a link, using a markdown feature that lets you put the url later